src/writer/normalize.rs

```diff
@@ -489,7 +489,7 @@ impl<World> CucumberQueue<World> {
     /// [`Feature`]: gherkin::Feature
     fn new_feature(&mut self, feat: Event<Arc<gherkin::Feature>>) {
         let (feat, meta) = feat.split();
-        drop(self.queue.insert(feat, FeatureQueue::new(meta)));
+        drop(self.fifo.insert(feat, FeatureQueue::new(meta)));
```

Comment:
@barafael not a fan of this particular change. It's more meaningful to emphasize we're _queuing_ something here, rather than bothering reader with exact semantics of the queue, which is "implementation detail".